packages/@aws-cdk/aws-cognito/test/user-pool.test.ts

```diff
@@ -1,17 +1,17 @@
 import '@aws-cdk/assert/jest';
 import { ABSENT } from '@aws-cdk/assert/lib/assertions/have-resource';
-import { Role } from '@aws-cdk/aws-iam';
+import * as iam from '@aws-cdk/aws-iam';
 import * as lambda from '@aws-cdk/aws-lambda';
-import { CfnParameter, Construct, Duration, Stack, Tag } from '@aws-cdk/core';
-import { Mfa, NumberAttribute, StringAttribute, UserPool, UserPoolIdentityProvider, UserPoolOperation, VerificationEmailStyle } from '../lib';
+import * as cdk from '@aws-cdk/core';
+import * as cognito from '../lib';
```

Comment:
Please revert these changes in this file. Use the same import style that already exists in the test.

Reply:
I see.

But it's still inconsistent.

```
import * as lambda from '@aws-cdk/aws-lambda';
```